versions/1/data_stream/manifest.spec.yml

```diff
@@ -87,6 +87,14 @@ spec:
       - beta
       examples:
       - beta
+    ilm_policy:
+      description: The name of an existing ILM (Index Lifecycle Management) policy
+      type: string
```

Comment:
As this is a string property, does it need a pattern definition here?

Reply:
no I do not believe it needs, it is the name of the ilm policy stored which is equivalent to a valid filename. I think this is guard enough.

Reply:
We introduced a strict validation for files and directories. If the ilm policy is part of the package, it's stored in some directory. You need to update/define a spec for that directory as well.

Reply:
is this a custom validation that needs to be written or it exist today?

Reply:
It's already defined, e.g.: https://github.com/elastic/package-spec/blob/master/versions/1/data_stream/spec.yml

Please review also the CONTRIBUTING doc.